app/services/transitions.ts

```diff
@@ -163,6 +167,14 @@ export class TransitionsService extends StatefulService<ITransitionsState> {
 
   enableStudioMode() {
     if (this.state.studioMode) return;
+    if (this.dualOutputService.views.dualOutputMode) {
+      this.notificationsService.actions.push({
+        message: $t('Cannot toggle Studio Mode in Dual Output Mode.'),
```

Comment:
i think you might have missed a translation here, unless this sentence is already in our system in which case pls disregard!

Reply:
I did indeed. Added